RHDEVDOCS-5795: Capitalization for 1.9 Repo table (#71947)
openshift/openshift-docs: modules/gitops-argo-cd-properties.adoc

|===
|*Name* |*Description* |*Default* | *Properties*
|`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
|`applicationSet` 
|`applicationSet` controller configuration options.
| `_<object>_`
a|* _<image>_ - The container image for the `ApplicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
  * _<version>_ - The tag to use with the `ApplicationSet` container image.
  * _<resources>_ - The container compute resources.
  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.

>   * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller `(--kubectl-parallelism-limit flag)`.

Reviewer: Just checking, ignore if no change is needed - are the parentheses something that you would include when using this flag, or is it used grammatically to introduce extra context to the description? Point being, if the parentheses are only used here in the description and not in the command, I wouldn't include it in the monospace styling:
```suggestion
  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
```

|`configManagementPlugins`    |Add a configuration management plugin.| `__<empty>__` |
|`controller`    |Argo CD Application Controller options.| `__<object>__`

> a|* _<processors.Operation>_ - The number of operation processors.

Reviewer 2: ```suggestion
a|* _<processors.operation>_ - The number of operation processors.
```

>   * _<processors.Status>_ - The number of status processors.

Reviewer 2: ```suggestion
  * _<processors.status>_ - The number of status processors.
```

  * _<resources>_ - The container compute resources.
  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.

>   * _<appSync>_ - AppSync is used to control the sync frequency of Argo CD applications

Reviewer: ```suggestion
  * _<appSync>_ - AppSync is used to control the sync frequency of Argo CD applications.
```

  * _<sharding.enabled>_ - Enable sharding on the Argo CD Application Controller component. This property is used to manage a large number of clusters to relieve memory pressure on the controller component.
  * _<sharding.replicas>_ - The number of replicas that will be used to support sharding of the Argo CD Application Controller.
  * _<env>_ - Environment to set for the application controller workloads.

|`disableAdmin`    |Disables the built-in admin user.|`false` |
|`gaTrackingID`    |Use a Google Analytics tracking ID.|`__<empty>__` |

> |`gaAnonymizeusers`    |Enable hashed usernames sent to google analytics.|`false` |

Reviewer 2: ```suggestion
|`gaAnonymizeUsers`    |Enable hashed usernames sent to google analytics.|`false` |
```

> |`HA`    |High availablity options.| `__<object>__`

Reviewer 2: ```suggestion
|`ha`    |High availablity options.| `__<object>__`
```

|`ingress`    |Ingress configuration options.| `__<object>__` |
|`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |

|`notifications`    |Notifications controller configuration options.|`__<object>__`
a|* `<enabled>` - The toggle to start the notifications-controller.
  * `<image>` - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
  * `<version>` - The tag to use with the Notifications container image.
  * `<resources>` - The container compute resources.
  * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.

|`repositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.| `__<empty>__` |
|`initialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.| `__<default_Argo_CD_Known_Hosts>__` |
|`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |

> |`oidCConfig` |The OIDC configuration as an alternative to Dex.|`__<empty>__` |

Reviewer 3: I think you have an extra capitalized C in oidc

|`nodePlacement` |Add the `nodeSelector` and the `tolerations`.|`__<empty>__` |

|`prometheus` |Prometheus configuration options.|`__<object>__`
a|* _<enabled>_ - Toggle Prometheus support globally for Argo CD.
  * _<host>_ - The hostname to use for Ingress or Route resources.
  * _<ingress>_ - Toggles Ingress for Prometheus.
  * _<route>_ - Route configuration options.
  * _<size>_ - The replica count for the Prometheus `StatefulSet`.

|`rbac` |RBAC configuration options.|`__<object>__`

> a|* _<defaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` config map. The name of the default role which Argo CD will fall back to, when authorizing API requests.

Reviewer 2: ```suggestion
a|* _<defaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` config map. The name of the default role which Argo CD falls back to when authorizing API requests.
```

  * _<policy>_ - The `policy.csv` property in the `argocd-rbac-cm` config map. CSV data containing user-defined RBAC policies and role definitions.
  * _<scopes>_ - The scopes property in the `argocd-rbac-cm` config map. Controls which OIDC scopes to examine during RBAC enforcement (in addition to sub scope).

|`redis` |Redis configuration options.|`__<object>__`

> a|* _<autoTLS>_ - Use the provider to create the Redis server's TLS certificate (one of: OpenShift). Currently only available for {OCP}.

Reviewer 2: ```suggestion
a|* _<autotls>_ - Use the provider to create the Redis server's TLS certificate (one of: OpenShift). Currently only available for {OCP}.
```

  * _<resources>_ - The container compute resources.
  * _<version>_ - The tag to use with the Redis container image.

|`resourceCustomizations` |Customize resource behavior.|`__<empty>__` |
|`resourceExclusions` |Completely ignore entire classes of resource group.|`__<empty>__` |
|`resourceInclusions` |The configuration to configure which resource group/kinds are applied.|`__<empty>__` |
|`server` |Argo CD Server configuration options.|`__<object>__`
a|* _<autoscale>_ - Server autoscale configuration options.
  * _<extraCommandArgs>_ - List of arguments added to the existing arguments set by the Operator.
  * _<grpc>_ - gRPC configuration options.
  * _<host>_ - The hostname used for Ingress or Route resources.
  * _<ingress>_ - Ingress configuration for the Argo CD server component.
  * _<insecure>_ - Toggles the insecure flag for Argo CD server.
  * _<resources>_ - The container compute resources.
  * _<replicas>_ - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `Autoscale` is enabled, `replicas` is ignored.
  * _<route>_ - Route configuration options.

>   * _<service.Type>_ - The `ServiceType` used for the service resource.

Reviewer 2: ```suggestion
  * _<service.type>_ - The `ServiceType` used for the service resource.
```

  * `<logLevel>` - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.

>   * `_<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.

Reviewer 3: you have an extra "_" in front of <logFormat>

  * _<env>_ - Environment to set for the server workloads.

|`sso` |Single Sign-on options.|`__<object>__`
a|* _<image>_ - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
  * _<keycloak>_ - Configuration options for Keycloak SSO provider.
  * _<dex>_ - Configuration options for Dex SSO provider.
  * _<provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
  * _<resources>_ - The container compute resources.

>   * _<verifyTLS>_ - Whether to enforce strict TLS checking when communicating with Keycloak service.

Reviewer 2: ```suggestion
  * _<verifyTLS>_ - This defines whether to enforce strict TLS checking when communicating with Keycloak service.
```

  * _<version>_ - The tag to use with the Keycloak container image.
|`statusBadgeEnabled` |Enable application status badge.|`true` |

|`tls` |TLS configuration options.|`__<object>__`
a|* _<ca.ConfigMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
  * _<ca.SecretName>_ - The name of the secret which contains the CA Certificate and Key.
  * _<initialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.

> |`userAnonyousEnabled` |Enable anonymous user access.|`true` |

Reviewer 2: ```suggestion
|`usersAnonymousEnabled` |Enable anonymous user access.|`true` |
```

|`version` |The tag to use with the container image for all Argo CD components.|Latest Argo CD version|

|`banner` |Add a UI banner message.|`__<object>__`
a|* _<banner.Content>_ - The banner message content (required if a banner is displayed).

>   * _<banner.URL.SecretName>_ - The banner message link URL (optional). 

Reviewer 2: ```suggestion
  * _<banner.url.SecretName>_ - The banner message link URL (optional). 
```
I did not see this property in the linked list, only `banner.url`: https://github.com/redhat-developer/gitops-operator/blob/master/bundle/manifests/argoproj.io_argocds.yaml#L7328 could you please check if the `secretName` part is still relevant? Thank you!
